main.py

```diff
@@ -232,7 +254,8 @@ def percent(part, whole):
 			# get all related bugs to job 
 			try:
 				bug_ids = blockers[job_name]['bz']
-				bugs = get_bugzilla(job_name, bug_ids)
+				#bugs = get_bugzilla(job_name, bug_ids)
+				bugs = list(map(all_bugs_dict.get,bug_ids))
```

Comment:
Shouldn't this replace line 275 as well?

Reply:
YES! sorry about the oversight